src/fxa-geodb.js

```diff
@@ -0,0 +1,79 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const db = '../db/cities-db.mmdb';
+const db_backup = '../db/cities-db.mmdb-backup';
+const ERRORS = require('../lib/errors');
+const Joi = require('joi');
+const maxmind = require('maxmind');
+const Promise = require('bluebird');
+
+function GeoDB(ip, time_stamp) {
+  'use strict';
```

Comment:
this code below should probably just use `return new Promise(function(resolve, reject) {`instead of calling `return Promise.reject({` everywhere